app/controllers/lti_controller.rb

```diff
@@ -11,25 +11,26 @@ class LtiController < ApplicationController
   layout 'embedded'
 
   def content_selection
-    # TODO: For testing purposes, the course, series and activity are chosen at
-    #       random. This should be replaced by a form that allows the user to
-    #       select something themself.
-    @series = Series.first
-    @course = @series.course
-    @activity = @series.activities.first
+    @supported = @lti_message.accept_types.include?(LTI::Messages::Types::DeepLinkingResponse::LtiResourceLink::TYPE)
+    courses = policy_scope(Course.all)
+    @grouped_courses = courses.group_by(&:year)
```

Comment:
Not using the `courses` variable (local scope), since it's not too long I'd just inline it

```suggestion
    @grouped_courses = policy_scope(Course.all).group_by(&:year)
```